ui/src/app/verify/result/page.tsx

```diff
@@ -4,31 +4,29 @@ import { Loader } from '@components'
 import { useStore, useVerifyProof } from '@hooks'
```

Comment:
On a second try it was faster: 
`==> Verify: 5768.683837890625 ms`

Probably the first time my browser was using lot of memory for other stuff.

Still, for 5s I think we must show a loader.
The first time I thought my browser froze because nothing was happening on the screen

Reply:
yeah we need to render this loading component. In my case it takes like you 5 s and sometimes 4 seconds. 
But I think its fine.